--- a52dec-0.7.5-cvs/liba52/configure.incl.orig	2004-03-03 10:47:28.000000000 +0100
+++ a52dec-0.7.5-cvs/liba52/configure.incl	2006-01-29 17:24:13.000000000 +0100
@@ -1,8 +1,8 @@
 AC_SUBST([LIBA52_CFLAGS])
 AC_SUBST([LIBA52_LIBS])
 
-dnl avoid -fPIC when possible
-AC_LIBTOOL_NON_PIC([LIBA52_CFLAGS="$LIBA52_CFLAGS -prefer-non-pic"])
+#dnl avoid -fPIC when possible
+#AC_LIBTOOL_NON_PIC([LIBA52_CFLAGS="$LIBA52_CFLAGS -prefer-non-pic"])
 
 AC_ARG_ENABLE([fixed],
     [  --enable-fixed          use fixed-point (integer-based) arithmetic])
